docs/howto/openstack/neutron/vpnaas.md

```diff
@@ -0,0 +1,741 @@
+    ```
+    Hit CTRL+C as soon as you see the status change you expect.
+        
+    Now, get on the right side of the connection (region `kna1`), 
+    optionally look for the name of the VPN connection to the left (in our 
+    example, that would be `vpn-conn-to-fra1`), and check its status:
+        
+    ```bash
+    openstack vpn ipsec site connection show vpn-conn-to-fra1 -c Status
+    ```
+        
+    ```plain
+    +--------+-------+
+    | Field  | Value |
+    +--------+-------+
+    | Status | DOWN  |
+    +--------+-------+
+    ```
+    
+    It turns out that this direction of the connection is also disabled.
+
+## Enabling an inactive connection
+
+You can easily enable an inactive site-to-site VPN connection. As soon
+as you enable such a connection, any two servers residing in different
+regions of the established connection will be again able to reach each
+other by private IP addresses. You will be able to verify that, e.g.,
+by pinging, as we demonstrate in the section regarding testing a
+site-to-site connection.
+
+=== "{{gui}}"
+    Currently, there is no way to enable an inactive connection from 
+    the {{gui}}. If you want to disable an inactive connection, please use 
+    the OpenStack CLI.
+=== "OpenStack CLI"
+    Make sure you hop on the side where you initially disabled the 
+    connection. According to the example scenario we described in the 
+    previous section, that would be the left side (region `fra1`), and the 
+    name of the disabled connection would be `vpn-conn-to-kna1`. Make sure 
+    the connection status is `DOWN`:
+        
+    ```bash
+    openstack vpn ipsec site connection show vpn-conn-to-kna1 -c Status
+    ```
+        
+    ```plain
+    +--------+-------+
+    | Field  | Value |
+    +--------+-------+
+    | Status | DOWN  |
+    +--------+-------+
+    ```
+        
+    Note that if you issued a similar command from the right side of 
+    the connection (region `kna1`), you would also get a `DOWN` status. 
+    Being on the left side, all you have to do to enable the inactive 
+    connection is type the following:
+        
+    ```bash
+    openstack vpn ipsec site connection set --enable vpn-conn-to-kna1
+    ```
+        
+    Check the connection status --- it should be `ACTIVE`:
+        
+    ```bash
+    openstack vpn ipsec site connection show vpn-conn-to-kna1 -c Status
+    ```
+        
+    ```plain
+    +--------+--------+
+    | Field  | Value  |
+    +--------+--------+
+    | Status | ACTIVE |
+    +--------+--------+
+    ```
+        
+    You get the same status by issuing a similar command from the right 
+    side.
+        
+    > Again, since you may have to wait several seconds before seeing 
+    the status change you expect, try something like this: 
+    ```bash
+    watch "openstack vpn ipsec site connection show vpn-conn-to-kna1 -c Status"
+    ``` 
+    Hit CTRL+C to stop watching.
```

Comment:
Again, here (outside the content tab) is where you restate the `ping` command, this time to verify that the connection is back.